test_data/test_project_facets/snooty.toml

```diff
@@ -0,0 +1,13 @@
+name = "test_data"
+title = "MongoDB title"
+canonical = "https://mongodb.com/docs/mongocli/install"
+
+[substitutions]
```

Comment:
remove substitutions for testing facets

Reply:
actually instead of adding this new projects, can we add facets.toml to existing test_project and make sure nothing fails? also can add new directive to `test_project/source/index.txt`